parts/k8s/kubernetescustomscript.sh

```diff
@@ -153,8 +153,11 @@ function installEtcd() {
     retrycmd_if_failure 10 1 5 sudo etcdctl member update $MEMBER ${ETCD_PEER_URL} || exit $ERR_ETCD_CONFIG_FAIL
 }
 
+function installDeps() {
+    apt_get_install 20 30 180 apt-transport-https ca-certificates iptables iproute2 socat util-linux mount ebtables ethtool init-system-helpers nfs-common ceph-common conntrack glusterfs-client ipset jq || exit $ERR_APT_INSTALL_TIMEOUT
```

Comment:
some packages are already installed in base image, install existing packages won't fail in this command, right?

Reply:
and would you also add package list source in comment:
`https://github.com/kubernetes/kubernetes/blob/master/build/debian-hyperkube-base/Dockerfile#L25-L44`

Reply:
To your 1st question, correct, if a package is already installed the command won’t exit non-zero